### ISSUE TITLE
Specific Cable For PX4 Debugger Ports

### ISSUE THREAD

**Issue by sam-rke** [PX4/PX4-Autopilot#15979]
Hi there,
I ordered a [Dronecode Probe](https://kb.zubax.com/display/MAINKB/Dronecode+Probe+documentation) for PX4 debugging purpose. I got a DCD-M cable with debugger but it is not compatible with PX4 debugging port.
I am using [PX4 Hex Cube Black Flight Controller](https://docs.px4.io/master/en/flight_controller/pixhawk-2.html).

Can Someone Suggest/Clarify me which cable I should order to be able to connect with PX4 debugging ports. Would be best if I can get the **Cable Name or Part Name** so that I can order that in India.

In below link, Is that correct one that I am looking for?
[Cable](https://www.digikey.com/en/products/detail/jst-sales-america-inc/A06SR06SR30K152A/6009379?s=N4IgTCBcDaIIIAYBsBlASs9BmBBpAjAKxhwgC6AvkA)

If yes what would be the part name to search it in Indian Sites.

Thankyou

**Comment by dagar** [comment]
The actual connector on the bottom of the Cube carrier board is a JST SUR.

**Comment by sam-rke** [comment]
> The actual connector on the bottom of the Cube carrier board is a JST SUR.

That's right @dagar  I saw that, it's on side of carrier.
My question is what is the cable name to be used with that specific ports?

**Comment by dagar** [comment]
Yes that's the correct one for the Dronecode probe. This might be useful if you don't want to make a custom cable (I haven't used it personally). https://1bitsquared.com/products/drone-code-debug-adapter

**Comment by sam-rke** [comment]
> Yes that's the correct one for the Dronecode probe. This might be useful if you don't want to make a custom cable (I haven't used it personally). https://1bitsquared.com/products/drone-code-debug-adapter

That's a different thing, I willl try for a custom cable as I already bought a Drone Code Probe....🤐
